index.js

```diff
@@ -61,6 +61,22 @@ express()
     }
 })
 
+<<<<<<< HEAD
+  .get('/addProduct', jsonParser, async (req, res) => {
+    try {
+      const client = await pool.connect();
+      const result = await client.query(`SELECT * FROM products where upper(prod_name)=upper('${req.query.searchParam}')AND user_ID IN ('0','1')`);
```

Comment:
@mdschnei should we update this to get the userID based on google login?

Reply:
It's completed!